apps/api/src/app/organization/ee.organization.controller.ts

```diff
@@ -75,6 +75,7 @@ export class EEOrganizationController {
 
   @Patch('/')
   @ExternalApiAccessible()
+  @UseGuards(RolesGuard)
```

Comment:
is this even possible with clerk do do it via this endpoint? I thought clerk managed these changes

Reply:
Yes it calls update on Clerk - it was originally left here for backwards compatibility, when the intent was to keep all the endpoints as they were before.

We should probably inspect usage and consider removing this.